codereview.java

```diff
@@ -0,0 +1,36 @@
+public class RuntimeIssuesDemo {
+
+    public static void main(String[] args) {
+        int[] numbers = {1, 2, 3};
+        System.out.println(numbers[3]); 
+
+        
+        String str = null;
+        System.out.println(str.length()); 
+
+        
+        int result = 10 / 0; 
```

Comment:
<b>code: int result = 10 / 0; - </b>
ArithmeticException might occur due to division by zero